packages/amplication-data-service-generator/src/server/create-grants.ts

```diff
@@ -92,6 +93,18 @@ export function createGrants(entities: Entity[], roles: Role[]): Grant[] {
           }
           break;
         }
+        case EnumEntityPermissionType.Public: {
+          for (const role of roles) {
+            grants.push({
+              role: role.name,
+              resource: entity.name,
+              action: actionToACLAction[permission.action],
+              /** @todo */
```

Comment:
What is this TODO?

Reply:
This is a copy & paste from the AllRoles case for now until we know how to handle this case. When I only `break` on this case the, I am getting errors on react admin after login